SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -28,36 +29,38 @@ public void AddButton()
         }
 
         /// <summary>
-        ///  txtbox Emp ID and insert stored ID
+        /// this is the second time on page where we...... 
+        /// search for emp id textbox and insert stored ID
+        /// then we click search button
         /// </summary>
         public void SetTxtBoxEmpID(string storeEmployeeID)
         {
             EditBoxSendKeysAndVerify(_employeeIDTxtBoxLocator, storeEmployeeID);
+            Click(_searchButtonLocator);
         }
 
-        public void SearchButton()
+        /// <summary>
+        ///  this is to verify ID # in search result
+        /// </summary>
+        public string verifyEmployeeId()
         {
-            Click(_searchButtonLocator);
+            return GetText(_actualIDLocator);
         }
 
         /// <summary>
-        ///  to verify ID # in search result
+        ///  this is to verify first name in search result
         /// </summary>
-        public void verifyEmployeeId(string employeeId)
+        public string verifyFirstNameResults()
         {
-            string expectedId = GetText(_expectedIDLocator);
-            Assert.AreEqual(employeeId, expectedId, "Employee ID is not match expected value.");
+            return GetText(_actualFirstNameLocator);
         }
 
         /// <summary>
-        ///  to verify name in searhc result
+        ///  this is to verify last name in search result
         /// </summary>
-        public void verifyName(string firstName, string lastName)
+        public string verifyLasttNameResults()
```

Comment:
This method name no longer matches what it's doing.

Reply:
same as first name. I renamed it VerifyLastNameResults so we can grab the text in the lastname search results to verify it does match

since all we are doing it grabbing text in the last name search results. should I just rename it.... 
LastNameSearchResults ?